Prog/HMDA_sum_P09.sas

```diff
@@ -0,0 +1,174 @@
+/**************************************************************************
+ Program:  HMDA_sum_P09.sas
+ Library:  HMDA
+ Project:  NeighborhoodInfo DC
+ Author:   Rob Pitingolo
+ Created:  9/11/18
+ Version:  SAS 9.4
+ Environment:  Windows 7
+ 
+ Description:  Create HMDA summary indicator file for Census tracts
+ (2000) from selected variables in HMDA files.
+
+ When updating: Make sure there is an updated version of %hud_inc_yyyy
+			    in the iPums macro library. 
+ 
+ Modifications:
+ 
+**************************************************************************/
+
+%include "L:\SAS\Inc\StdLocal.sas"; 
+
+** Define libraries **;
+%DCData_lib( HMDA )
+%DCData_lib( ipums )
+
+%let start_yr = 2009;
+%let end_yr = 2016;
+
+%let revisions = New file ;
+
+%let tr2000data = hmda2009tr hmda2010tr hmda2011tr;
+%let tr2010data = hmda2012tr hmda2013tr hmda2014tr hmda2015tr hmda2016tr;
+
+
+
+/*Create tract-level summary file for each year of data */
+%hmda_tracts_new (&start_yr.,&end_yr.);
+
+
+/* Merge 2000-tract year files */
+data hmda_long00_tr00;
+	merge &tr2000data.;
+	by geo2000;
+run;
+
+proc contents data = hmda_long00_tr00 out = vars2000;
+run;
+
+data vars2000p;
+	set vars2000;
+	prefix = upcase(substr(name,1,3));
+run;
+
+proc sort data = vars2000p; by name; run;
+
+proc sql noprint;
+select name
+into :count_vars separated by " "
+from vars2000p (where=(prefix="NUM"));
+quit;
+
+proc sql noprint;
+select name
+into :median_vars separated by " "
+from vars2000p (where=(prefix="MED"));
+quit;
+
+/* Create summary files from 2000-tract files */
+%Create_all_summary_from_tracts( 
+  lib=work,
+  data_pre=hmda_long00,
+  data_label=,
+  count_vars=&count_vars., 
+  prop_vars=&median_vars., 
+  calc_vars=, 
+  calc_vars_labels=,
+  tract_yr=2000,
+  register=,
+  finalize=y,
```

Comment:
Do we want to save these as permanent data sets? It seems like it is sufficient to keep only the final summary files. In which case, this should be _finalize=n_.  

But, now I'm realizing this parameter no longer does anything. So, we'll have to fix the %Create_summary_from_tracts() macro in the General library. I've created a [new issue](https://github.com/NeighborhoodInfoDC/General/issues/67) for this.